src/helpers/Constants.ts

```diff
@@ -131,7 +131,7 @@ const constants = {
   },
   /* Dimensions */
   removeDimensionsEventListener: (callback: any) => {
-    Dimensions.removeEventListener('change', callback);
+    Dimensions.remove('change', callback);
```

Comment:
Our demo app still uses RN64 which doesn't support this method. 
That means changing it is a breaking change. 
I'd consider supporting both RN APIs for now

Reply:
Updated RN version, forgot it.

Reply:
That's actually want not my intention :( 

Having this change enforce our users to migrate to RN65, which is not a trivial change for people. 
So to avoid that, and to allow both users with `<0.65` and `>=0.65` to use rnuilib I suggest doing the following

```suggestion
    if (Dimensions.remove) {
      Dimensions.remove('change', callback);
    } else {
      Dimensions.removeEventListener('change', callback);
    }
```